CommunityFridgeMapApi/functions/dev/load_fridge_data/app.py

```diff
@@ -0,0 +1,57 @@
+import os
+import json
+import logging
+from botocore.exceptions import ClientError
+from db import get_ddb_connection
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+FRIDGE_DATA = [
+                {
+                    'PutRequest': {
+                        'Item': {
+                            'state': {'S': 'NY'},
+                            'fridge_name': {'S': 'The Friendly Fridge'},
+                            'address': {'S': '1046 Broadway Brooklyn, NY 11221'},
+                            'instagram': {'S': 'https://www.instagram.com/thefriendlyfridge/'}
+                        }
+                    }
+                },
+                {
+                    'PutRequest': {
+                        'Item': {
+                            'state': {'S': 'NY'},
+                            'fridge_name': {'S': '2 Fish 5 Loaves Fridge'},
+                            'address': {'S': '63 Whipple St, Brooklyn, NY 11206'},
+                            'instagram': {'S': 'https://www.instagram.com/2fish5loavesfridge/'}
+                        }
+                    }
+                },
+            ]
+
+
+FRIDGE_CHECK_IN_DATA = []
+FRIDGE_HISTORY_DATA = []
+
+def lambda_handler(event: dict, context: 'awslambdaric.lambda_context.LambdaContext') -> dict:
```

Comment:
this file will eventually get deleted. It's purpose is to be able to test that we can successfully write to the db